pkg/daemon/osrelease.go

```diff
@@ -44,6 +44,15 @@ func (os OperatingSystem) IsLikeTraditionalRHEL7() bool {
 	return os.VersionID == "7"
 }
 
+// IsLikeTraditionalRHEL8 is true if the OS is traditional RHEL8 or CentOS8:
```

Comment:
Is this func only used in the below unit test?

Reply:
It's only used there because we don't know of any ways RHEL8 is broken yet (all of the RHEL7 decisions were because something was "wrong" with RHEL7 relative to CoreOS, , but at some point if we have to do something RHEL8-specific (bug, feature, whatever) we'll have to add it. 

I can take it out and hold it if we don't do "speculative code" that we don't 100% have a use for right now.

Reply:
Yes, please remove. :+1:

Reply:
agree with Kirsten, it doesn't make sense to add a function in advance if there is no usecase today. It can be added as part of fix if required in future.